service/worker/pernamespaceworker.go

```diff
@@ -452,21 +452,25 @@ func (w *perNamespaceWorker) startWorker(
 
 	var sdkoptions sdkworker.Options
 
-	// copy from dynamic config
-	sdkoptions.MaxConcurrentActivityExecutionSize = dcOptions.MaxConcurrentActivityExecutionSize
+	// copy from dynamic config. apply explicit defaults for some instead of using the sdk
+	// defaults so that we can multiply below.
+	sdkoptions.MaxConcurrentActivityExecutionSize = util.Coalesce(dcOptions.MaxConcurrentActivityExecutionSize, 1000)
 	sdkoptions.WorkerActivitiesPerSecond = dcOptions.WorkerActivitiesPerSecond
-	sdkoptions.MaxConcurrentLocalActivityExecutionSize = dcOptions.MaxConcurrentLocalActivityExecutionSize
+	sdkoptions.MaxConcurrentLocalActivityExecutionSize = util.Coalesce(dcOptions.MaxConcurrentLocalActivityExecutionSize, 1000)
 	sdkoptions.WorkerLocalActivitiesPerSecond = dcOptions.WorkerLocalActivitiesPerSecond
-	sdkoptions.MaxConcurrentActivityTaskPollers = util.Max(2, dcOptions.MaxConcurrentActivityTaskPollers)
-	sdkoptions.MaxConcurrentWorkflowTaskExecutionSize = dcOptions.MaxConcurrentWorkflowTaskExecutionSize
-	sdkoptions.MaxConcurrentWorkflowTaskPollers = util.Max(2, dcOptions.MaxConcurrentWorkflowTaskPollers)
+	sdkoptions.MaxConcurrentActivityTaskPollers = util.Coalesce(dcOptions.MaxConcurrentActivityTaskPollers, 2)
+	sdkoptions.MaxConcurrentWorkflowTaskExecutionSize = util.Coalesce(dcOptions.MaxConcurrentWorkflowTaskExecutionSize, 1000)
+	sdkoptions.MaxConcurrentWorkflowTaskPollers = util.Coalesce(dcOptions.MaxConcurrentWorkflowTaskPollers, 2)
```

Comment:
I'm assuming setting `MaxConcurrentActivityTaskPollers` and `MaxConcurrentWorkflowTaskPollers` to `1` is okay? Asking since before this change the lowest possible value was `2`.

Reply:
It's not okay. I'll add back the max with 2